code/provision/playbooks/roles/docker/defaults/main.yml

```diff
@@ -1,11 +1,8 @@
 ---
-docker_edition: "ce"
-docker_channel: "stable"
-
-docker_version: "17.03.2"
+docker_version: "18.06.2~ce~3-0~ubuntu"
```

Comment:
Are you installing Docker form the Ubuntu Apt Repository or from Docker.io?

Reply:
From docker.io